### ISSUE TITLE
Feature: choose typeface for list mode and preview

### ISSUE THREAD

**Issue by schriftgestalt** [brentsimmons/NetNewsWire#888]
I liked the option in NNW 4 to set the typeface.

**Comment by macdaddy1944** [comment]
Yes, would like to be able to set font and size for the sidebar

**Comment by SomeCallMeDave** [comment]
I agree - the font is just too small for me to read comfortably and there is (afaict) no way to change it.

**Comment by vincode-io** [comment]
Coming in version 6.0 of NetNewsWire, we will respect the Sidebar settings that can be changed in System Preferences in the General section.  Changing this to the Large setting will make all your application Sidebars bigger and more readable.
